node/p2p/identity.go

```diff
@@ -2,21 +2,49 @@ package p2p
 
 import (
 	"crypto/rand"
+	"errors"
 
 	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/libp2p/go-libp2p-core/peerstore"
+
+	"github.com/celestiaorg/celestia-node/libs/keystore"
 )
 
+// TODO(Josh): What should the real name for this be?
+const keyName = "test-key"
```

Comment:
`p2p-key` is fine or did it cause problems?

Reply:
@jbowen93 check this when u have time.